createNPMPackage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+yarn install --immutable
+
+if ! CURRENT_VERSION=$(node scripts/set-version.js); then
+  exit 1
+fi
+
+yarn build
+
+npm pack
+
+node scripts/set-version.js "$CURRENT_VERSION" >/dev/null
```

Comment:
If we pack, before changing version in `package.json` we'll use old version for the artifact, right?